terraform/modules/rds/database.tf

```diff
@@ -0,0 +1,34 @@
+#tfsec:ignore:aws-rds-specify-backup-retention:TODO CLDC-2261
+#tfsec:ignore:aws-rds-enable-performance-insights:TODO CLDC-2260 if necessary
+#tfsec:ignore:AVD-AWS-0176:iam authentication not suitable as tokens only last 15minutes, password authentication preferred
```

Comment:
There seems to be a slight bug with `tfsec` here in that didn't output out a "worded id" of the rule it was flagging, so we've had to use a different reference (`AVD-AWS-0176`) to ignore it. [Here's a bug report](https://github.com/aquasecurity/tfsec/issues/1935) of a similar issue. One option could be to downgrade tfsec from `1.28.1` to `1.28.0` if we see many more like this in future.